x-pack/plugins/ml/public/datavisualizer/selector/datavisualizer_selector.js

```diff
@@ -25,14 +26,23 @@ import { isFullLicense } from '../../license/check_license';
 function startTrialDescription() {
   return (
     <span>
-      To experience the full Machine Learning features that a {' '}
+      <FormattedMessage
+        id="xpack.machineLearning.datavisualizer.startTrial.fullMachineLearningFeaturesDescription"
+        defaultMessage="To experience the full Machine Learning features that a "
```

Comment:
Please don't break one sentence in separate messages.
```suggestion
        defaultMessage="To experience the full Machine Learning features that a {platinumSubscriptionLink} offers, start a 30-day trial."
```